contracts/package.json

```diff
@@ -8,10 +8,10 @@
   "devDependencies": {
     "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
     "@nomicfoundation/hardhat-toolbox": "^2.0.2",
-    "hardhat": "^2.16.0",
```

Comment:
I wasn't sure where to leave a comment about the README, so I'm posting it here.

It seems that running `npx hardhat node` is no longer a prerequisite for executing `npx hardhat test` when testing contracts. Could we consider moving the `npx hardhat node` command line to the Deploying section in the README?

https://github.com/summa-dev/summa-solvency/tree/halo2-verifier/contracts#testing